DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/DependencyInjection/AutofacUIModule.cs

```diff
@@ -78,9 +78,6 @@ protected override void Load(ContainerBuilder builder)
     // register
     builder.RegisterType<RevitDocumentStore>().SingleInstance();
```

Comment:
If there can only ever be one document store, we should justify in a comment why is that to prevent someone changing it without context.